I got the following error running the make in `depends` folder

```
$ make HOST=arm-unknown-linux-gnueabihf NO_QT=1 # NO_WALLET=1 NO_UPNP=1
Building boost...
/home/drak/bitcoin/depends/work/build/arm-unknown-linux-gnueabihf/boost/1_59_0-16ec5a90819/tools/build/src/tools/gcc.jam:83: in gcc.init from module gcc
error: toolset gcc initialization:
error: provided command 'arm-unknown-linux-gnueabihf-g++' not found
error: initialized from /home/drak/bitcoin/depends/work/build/arm-unknown-linux-gnueabihf/boost/1_59_0-16ec5a90819/user-config.jam:1
/home/drak/bitcoin/depends/work/build/arm-unknown-linux-gnueabihf/boost/1_59_0-16ec5a90819/tools/build/src/build/toolset.jam:43: in toolset.using from module toolset
/home/drak/bitcoin/depends/work/build/arm-unknown-linux-gnueabihf/boost/1_59_0-16ec5a90819/tools/build/src/build/project.jam:1052: in using from module project-rules
/home/drak/bitcoin/depends/work/build/arm-unknown-linux-gnueabihf/boost/1_59_0-16ec5a90819/user-config.jam:1: in modules.load from module user-config
/home/drak/bitcoin/depends/work/build/arm-unknown-linux-gnueabihf/boost/1_59_0-16ec5a90819/tools/build/src/build-system.jam:249: in load-config from module build-system
/home/drak/bitcoin/depends/work/build/arm-unknown-linux-gnueabihf/boost/1_59_0-16ec5a90819/tools/build/src/build-system.jam:385: in load-configuration-files from module build-system
/home/drak/bitcoin/depends/work/build/arm-unknown-linux-gnueabihf/boost/1_59_0-16ec5a90819/tools/build/src/build-system.jam:524: in load from module build-system
/home/drak/bitcoin/depends/work/build/arm-unknown-linux-gnueabihf/boost/1_59_0-16ec5a90819/tools/build/src/kernel/modules.jam:295: in import from module modules
/home/drak/bitcoin/depends/work/build/arm-unknown-linux-gnueabihf/boost/1_59_0-16ec5a90819/tools/build/src/kernel/bootstrap.jam:139: in boost-build from module
/home/drak/bitcoin/depends/work/build/arm-unknown-linux-gnueabihf/boost/1_59_0-16ec5a90819/boost-build.jam:17: in module scope from module

funcs.mk:238: recipe for target '/home/drak/bitcoin/depends/work/build/arm-unknown-linux-gnueabihf/boost/1_59_0-16ec5a90819/./.stamp_built' failed
make: *** [/home/drak/bitcoin/depends/work/build/arm-unknown-linux-gnueabihf/boost/1_59_0-16ec5a90819/./.stamp_built] Error 1
```

--- a/doc/build-unix.md
+++ b/doc/build-unix.md
@@ -248,3 +248,26 @@ Additional Configure Flags
 A list of additional configure flags can be displayed with:
 
     ./configure --help
+
+
+
+ARM Cross-compilation
+-------------------
+These steps can be performed on, for example, an Ubuntu VM. The depends system
+will also work on other Linux distributions, however the commands for
+installing the toolchain will be different.
+
+First install the toolchain:
+
+    sudo apt-get install g++-arm-linux-gnueabihf
+
+To build executables for ARM:
+
+    cd depends
+    make HOST=arm-linux-gnueabihf NO_QT=1
+    cd ..
+    ./configure --prefix=$PWD/depends/arm-linux-gnueabihf --enable-glibc-back-compat --enable-reduce-exports LDFLAGS=-static-libstdc++
+    make
+
+
+For further documentation on the depends system see [README.md](../depends/README.md) in the depends directory.

